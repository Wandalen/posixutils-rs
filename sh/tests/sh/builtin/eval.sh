eval
echo $?

eval --
echo $?

eval echo hello
echo $?

eval -- echo hello
echo $?

eval echo hello \; echo world
echo $?

eval true '&&' echo correct
echo $?

eval false
echo $?

eval echo test > $TEST_WRITE_DIR/eval.txt
echo $?
cat $TEST_WRITE_DIR/eval.txt
rm $TEST_WRITE_DIR/eval.txt