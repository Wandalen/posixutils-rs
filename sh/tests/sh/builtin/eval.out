0
0
hello
0
hello
0
hello
world
0
correct
0
1
0
test
